src/crangler/Makefile

```diff
@@ -0,0 +1,36 @@
+SRC = c_defines.cpp \
```

Comment:
Somewhat arbitrary place for comment: we're missing a CMake configuration for this new tool.